backend/infrahub/git/integrator.py

```diff
@@ -528,6 +530,22 @@ async def import_schema_files(self, branch_name: str, commit: str, config_file:
                 f"schema '{schema_file.identifier}' loaded successfully!", repository=self.name, commit=commit
             )
 
+        max_wait_time = 60
+        if response.schema_updated:
```

Comment:
I think it would be good to include this function in the SDK directly
I'm sure it will be useful in other situations

Reply:
I would prefer to decouple things as much as possible. Generated GraphQL queries sounds great but since we don't have a committed plan for it we shouldn't hold an important feature. 
Also, it feels like the public method to `wait_for_schema_to_converge` (or similar) shouldn't really change if we are using a generated GraphQL query internally or not.

Reply:
I opened this PR in the SDK: https://github.com/opsmill/infrahub-sdk-python/pull/155 Once merged I'll update this PR with the changes. I'm pretty sure we'll want to change that later, but could be for SDK 2.0 or later.